torchchat/usages/browser.py

```diff
@@ -1,39 +1,123 @@
+import base64
+import logging
 import time
+from pathlib import Path
+
 import streamlit as st
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
 from openai import OpenAI
 
 st.title("torchchat")
 
 start_state = [
     {
         "role": "system",
-        "content": "You're an assistant. Answer questions directly, be brief, and have fun.",
+        "content": "You're a helpful assistant - have fun.",
```

Comment:
Why do we need to update the system prompt? Any source of truth for that?

Reply:
No source of truth - I just made it shorter to reduce the amount of pre-filling the model needs to do. I have another PR that allows users to change this.